src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -175,3 +181,51 @@ export const getLastScheduledDayDiff = (
     new Date(mostRecentScheduleDate).getTime();
   return Math.abs(Math.ceil(daysDifference / (1000 * 3600 * 24)));
 };
+
+/**
+ * Formats an excerpt using rules based on the excerpt's language.
+ * @param language the language the passed excerpt is in
+ * @param excerpt excerpt to format
+ * @returns formatted excerpt
+ */
+export const applyExcerptFormattingByLanguage = (
+  language: CorpusLanguage,
+  excerpt: string,
+): string => {
+  // if excerpt is English, apply quotes formatting for EN
+  if (language === CorpusLanguage.En) {
+    applyCurlyQuotes(excerpt);
+  }
+  // if excerpt is German, apply German quotes/dashes formatting
+  if (language === CorpusLanguage.De) {
+    return applyQuotesDashesDE(excerpt) as string;
+  }
+  // apply EN formatting rules on excerpt for other languages for now
+  else {
+    return applyCurlyQuotes(excerpt);
+  }
```

Comment:
just a little cleanu. this function could be simplified to:

```
// DE is the only special case for now
if (langauge === CorpusLanguage.De) {
  return applyQuotesDashesDE(excerpt) as string;
// all other languages get the EN treatment
} else {
  return applyCurlyQuotes(excerpt);
}
```

same pattern for the function below.